.github/workflows/storybook.yml

```diff
@@ -0,0 +1,46 @@
+name: Chromatic Storybook
+
+on: push
+
+jobs:
+  deployment:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2.4.0
+        with:
+          # Chromatic requires full git history.
+          fetch-depth: 0
+
+      - name: "Install & Compile: common"
+        working-directory: common
+        run: |
+          npm install
+          npm run h2-build
+          npm run codegen
+
+      - name: "Install & Compile: admin"
+        working-directory: admin
+        run: |
+          npm install
+          npm run h2-build
+          npm run codegen
+
+      - name: "Install & Compile: talentsearch"
+        working-directory: talentsearch
+        run: |
+          npm install
+          npm run h2-build
+          npm run codegen
+
+      - name: "Publish Storybook: all"
+        uses: chromaui/action@v1
+        env:
+          MERGE_STORYBOOKS: true
+        with:
+          workingDir: admin
+          projectToken: 544cb0da12a4
```

Comment:
Recommend replacing with a project token generated by someone with access to repo, though no required.

Left as-is, should still publish storybook and print its link in CI logs, just won't be able to update the PR commit status with a convenient "Details" link. Link format for this project token is always `https://<branch>--61de6c057fec2d003a9bc22f.chromatic.com` e.g. https://publish-storybook--61de6c057fec2d003a9bc22f.chromatic.com